webapp/src/templates/partials/reports_content.html

```diff
@@ -64,14 +64,16 @@
               {{field.label}}
               <span class="fa fa-cogs" ng-show="field.generated" translate translate-attr-title="Generated report field"></span>
             </label>
-            <span ng-if="field.hasUrl"><a ui-sref="contacts.detail({ id: reportsContentCtrl.summaries[$index].subject._id || reportsContentCtrl.summaries[$index].fields.patient_uuid })">{{field.value}}</a></span>
-            <span ng-if="!field.hasUrl">{{field.value}}</span>
+            <span ng-if="field.target && field.target.filter"><a ng-click="reportsContentCtrl.search(field.target.filter)">{{field.value}}</a></span>
+            <span ng-if="field.target && field.target.url"><a ui-sref="{{field.target.url.route}}({{field.target.url.params}})">{{field.value}}</a></span>
```

Comment:
The sad part about this is that when I click on this link, my report gets "deselected". 
https://github.com/medic/cht-core/issues/4825

Reply:
Yeah, it's a little counter intuitive, but consistent. I think we stick with it for now but may need to review the solution to 4825.

I think the "proper" solution would be to only clear the RHS if the report/contact isn't available in the LHS. But that may be a bit weird given the search can be slow, so does the RHS just sit there and then disappear a few seconds later, or do we deselect it then reselect it?

I could raise an issue to get design input?

Reply:
I agree with your idea of the proper way! let's get design input!

Reply:
Raised: https://github.com/medic/cht-core/issues/6367